core/src/main/java/io/aiven/klaw/service/EnvsClustersTenantsControllerService.java

```diff
@@ -926,6 +925,8 @@ private void removeAssociationWithKafkaEnv(EnvTag envTag, String envId, int tena
               .getEnvDetails(existingEnv.getAssociatedEnv().getId(), tenantId);
       linkedEnv.setAssociatedEnv(null);
       manageDatabase.getHandleDbRequests().addNewEnv(linkedEnv);
+      // remove from cache
```

Comment:
```suggestion
      // add to cache
```
probably you mean this comment